cljfmt/resources/cljfmt/indents/fuzzy.clj

```diff
@@ -1,2 +1,2 @@
-{#"^def"   [[:inner 0]]
- #"^with-" [[:inner 0]]}
+{#"^def(?!ault)(?!late)(?!er)"   [[:inner 0]]
```

Comment:
You can get rid of the extra spaces between `er)"` and `[[:in`

Reply:
Good catch! Amended